distributed/comm/tcp.py

```diff
@@ -160,10 +160,10 @@ def _read_extra(self):
         pass
 
     def _get_finalizer(self):
-        def finalize(stream=self.stream, r=repr(self)):
-            if not stream.closed():
+        def finalize(comm=self.stream, r=repr(self)):
+            if not comm.closed():
                 logger.warning("Closing dangling stream in %s" % (r,))
-                stream.close()
+                comm.close()
```

Comment:
This is actually a different kind of stream (Torando IOStream rather than Dask stream) and we should probably leave it as-is.